frontend/src/component/admin/serviceAccounts/ServiceAccountsTable/ServiceAccountModal/ServiceAccountTokens/ServiceAccountTokens.tsx

```diff
@@ -228,7 +218,7 @@ export const ServiceAccountTokens = ({
 
     const { headerGroups, rows, prepareRow, setHiddenColumns } = useTable(
         {
-            columns,
+            columns: columns as any[],
```

Comment:
Scouting: This had a TS error. The diff above is caused by the removal of `as Column<IPersonalAPIToken>[]` - It should otherwise be the same. `columns: columns as any[]` is more consistent with what we usually do anyways.